docs/sdk-keys/target-apps.mdx

```diff
@@ -0,0 +1,63 @@
+
+:::info
+NOTE: Before using target apps as a solution, consider if you can instead use separate projects for your statsig integrations.  Note that gates/experiments/metrics are restricted to a single Statsig project, so if you need to share any of those across apps or services, its best to use a Target App instead. But if your services or apps always have distinct gates/experiments/metrics, using a separate project might be a better solution.
+:::
+
+If you have an existing Statsig project with multiple SDK integrations, Statsig can help you bootstrap your Target Apps by tagging existing entities which have recorded checks from that specific SDK.  This would help you at least segment your project by which sdks are accessing them
+
+## Example
+
+### Project Setup
+
+Start by going to Project Settings > Keys and Environments.  There you will find a "Target Apps" section.
+
+![Screen Shot 2023-07-26 at 4 18 46 PM](https://github.com/statsig-io/js-client/assets/74584483/6a2d9fa5-b3f9-4fd3-91e5-3487ebd02618)
+
+Create a target app, or use one that is suggested based on your current SDK exposure logs.
+
+![Screen Shot 2023-07-26 at 5 17 53 PM](https://github.com/statsig-io/docs/assets/74584483/719a399d-66f2-48a5-bf42-20f383758c7f)
+
+### Associating a Feature Gate with a Target App
+
+If you wish to verify the suggestions you added in the initial setup, or add additional entities to the target app, you can do so from that entity itself.
+
+For example, navigate to the Feature Gates tab, and then select a specific gate.
+
+In the header for that gate, you view and edit the target app(s)
+
+![Screen Shot 2023-07-26 at 4 19 21 PM](https://github.com/statsig-io/docs/assets/74584483/9ef90688-7aff-41b6-b495-f64e692774be)
+
+Repeat this flow for the Dynamic Configs and Experiments that are used by this Target App.
+
+### Associating an SDK Key with a Target App
+
+Finally, return to your Project Settings > Keys and Environments page. You're ready to restrict a certain key to a certain target app.
+
+Select the SDK Key you wish to restrict to the set of Target App entities.
+
+Under Actions, trigger the dropdown menu
+
+![Screen Shot 2023-07-26 at 5 23 17 PM](https://github.com/statsig-io/docs/assets/74584483/9e44aa27-4034-45c9-a5fb-94d613c7d2ab)
+
+Then select the target app(s) to apply to that key.
+
+![Screen Shot 2023-07-26 at 5 23 21 PM](https://github.com/statsig-io/docs/assets/74584483/e9559312-71af-4d70-85b4-a618e76c0008)
+
+:::warn
+NOTE: If you are using a server SDK to bootstrap your client integration, you need the server SDK Key to have access to both the client and server definitions.  This means you should associate both the client and server target apps with that SDK Key.  You will have to pass in the client key for which you wish to initialize when calling getClientInitializeResponse - this behavior is only supported in statsig-node v5.8.2+.
+:::
```

Comment:
These are great @tore-statsig, thanks for drafting! 

One section that might be worth adding is "Debug Tools" and just note that we surface the Target App in the Exposure Stream log detail, as well as highlight if any exposures are coming in from a Target App not currently added to that entity (shows up as the little triangle warning icon in the Exposure Stream associated w/ that exposure).